src/domains/aca-app/api/aca/v1/_openapi.json.tpl

```diff
@@ -0,0 +1,218 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "version": "0.0.1",
+    "title": "Pagopa ACA",
+    "description": "pagoPA ACA microservice pagoPA ACA microservice contains the api to allow the creation of a new debt position.",
+    "contact": {
+      "name": "pagoPA - Touchpoints team"
+    }
+  },
+  "externalDocs": {
+    "url": "https://pagopa.atlassian.net/wiki/spaces/PPR/pages/649265163/ACA+-+Dati+in+input+API",
+    "description": "Technical specifications"
+  },
+  "servers": [
+    {
+      "url": "https://${hostname}/pagopa-aca-service"
+    }
+  ],
+  "tags": [
+    {
+      "name": "ACA",
+      "description": "Api's for performing a debt position census",
+      "externalDocs": {
+        "url": "https://pagopa.atlassian.net/wiki/spaces/PPR/pages/649265163/ACA+-+Dati+in+input+API",
+        "description": "Technical specifications"
+      }
+    }
+  ],
+  "paths": {
+    "/paCreatePosition": {
+      "post": {
+        "tags": [
+          "ACA"
+        ],
+        "operationId": "newDebtPosition",
+        "summary": "Create a new debt position",
+        "description": "Create a new debt position.",
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/NewDebtPositionRequest"
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
```

Comment:
Should here be also 200 status code used for debt position deletion/update requests?

Reply:
Fixed with be0ebd08a30e1463f6f7e0df0803d9d46b88ae25 🙂